packages/fx-core/src/plugins/resource/bot/functionsHostedBot/plugin.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Add code owners for `functionHostedBot/`, to include both teams' members.

Reply:
Added